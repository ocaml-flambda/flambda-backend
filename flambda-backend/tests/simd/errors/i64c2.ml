open Stdlib

type t = int64x2

external const2 : int64 -> int64 -> t = "" "caml_int64x2_const2"
        [@@noalloc] [@@unboxed] [@@builtin]

let _ = const2 0L (Sys.opaque_identity 1L)
