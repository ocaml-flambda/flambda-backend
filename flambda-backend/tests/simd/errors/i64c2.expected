File "i64c2.ml", line 1:
Error: Did not find constant int64 arguments for caml_int64x2_const2
